plugin.js

```diff
@@ -40,6 +40,13 @@ function promise () {
   return obj
 }
 
+function loadPluginNextTick (toLoad, cb) {
```

Comment:
Can you add a comment explaining why this is necessary?

Reply:
update